fsspec/implementations/tests/test_local.py

```diff
@@ -597,7 +597,7 @@ def test_copy_errors(tmpdir):
 
 
 def test_transaction(tmpdir):
-    file = (tmpdir / "test.txt").as_posix()
+    file = str(tmpdir / "test.txt")
```

Comment:
Would this do the right thing on Windows?

Reply:
I can confirm that on windows we need to use as_posix path representation. I just recently fixed such issue in my code.